src/Runner.Worker/action_yaml.json

```diff
@@ -180,6 +181,7 @@
                 "matrix",
                 "job",
                 "runner",
+                "vars",
```

Comment:
why we need `vars` here?

Reply:
+1 we dont need in actions yaml file, since we support `vars` only in workflow file.

Reply:
Thanks for mentioning this. I'll remove the changes from this file.